### DISCUSSION TITLE
query about data export

### DISCUSSION THREAD

**Discussion by fcobarahona** [Open-Systems-Pharmacology/Forum#1087] [Questions & Problems]
how can I extract the individual concentration profiles for a population simulation?

**Comment by msevestre** [comment]
Hi @fcobarahona 
Most options can be accessed via the popup menu on the simulation
In your case,  Export Results to CSV
![image](https://user-images.githubusercontent.com/1041237/150386106-8329309c-6dfd-4e0c-8117-2b7c5f5d7121.png)
